torch/testing/_internal/common_methods_invocations.py

```diff
@@ -14951,10 +14951,6 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                # JIT has issue when op is passed as lambda
                # AssertionError: JIT Test does not execute any logic
                DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),
-               # Fails due to a limitation of gradgradcheck
-               # https://github.com/pytorch/pytorch/issues/59137
-               DecorateInfo(unittest.expectedFailure, 'TestGradients', 'test_fn_gradgrad'),
-               DecorateInfo(unittest.expectedFailure, 'TestGradients', 'test_inplace_gradgrad'),
```

Comment:
cc @soulitzer, these tests started passing after I added a derivative formula for `zeros_like`. Just double checking - does that sound reasonable? (If it is I can close the linked issue)